### ISSUE TITLE
Loosen the version constraint of persisting_theory to avoid dependency

### ISSUE THREAD

**Issue by NeolithEra** [EliotBerriot/django-dynamic-preferences#182]
Hi, _**django-dynamic-preferences**_ locked the version constraint of _**persisting_theory**_ as **_persisting_theory==0.2.1_**, which leads to a troubling scenario that its direct downstream project **_[omaha-server,'socialhome', 'django-danceschool']_** has to lock **_persisting_theory_**.

Could you please loosen the version constraint of _**persisting_theory**_?
Benefit of this is that users using both of _**[omaha-server,'socialhome', 'django-danceschool']**_ and _**persisting_theory**_ can upgrade their third party libraries in a timely manner to reduce technical debts.

### Solution

The dependency trees of your project and affected downstream projects are shown as follows.
Taking the version constraints of upstream and downstream projects into comprehensive consideration, you can

1.	Loosen persisting_theory==0.2.1 to be persisting_theory>=0.2.1.
2.	Loosen persisting_theory==0.2.1 to be persisting_theory==*.
3.	Try to add an upper bound for persisting_theory’ version constraint, according to your compatibility.

@EliotBerriot   Please let me know your choice. I can submit a PR to fix this issue.

Thanks for your attention.
Best,
Neolith

**Comment by NeolithEra** [comment]
### Dependency tree-----
```
django-dynamic-preferences-develop
| +-django(version range:>=1.11)
| +-persisting-theory(version range:==0.2.1)
| +-six(version range:*)
```




_**Direct downstream project:**_

```
omaha-server-master
.
.
.
| +-django-debug-toolbar(version range:==1.5)
| | +-django(version range:>=1.8)
| | | +-pytz(version range:*)
| | +-sqlparse(version range:>=0.2.0)
| +-django-dynamic-preferences(version range:==1.7.0)
| | +-django(version range:>=1.11)
| | | +-pytz(version range:*)
| | +-persisting-theory(version range:==0.2.1)
| | +-six(version range:*)
| +-persisting-theory(version range:==0.2.1)
.
.
.
```


```
Socialhome
.
.
.
| +-django-dynamic-preferences(version range:==1.7.0)
| | +-django(version range:>=1.8)
| | | +-pytz(version range:*)
| | +-persisting-theory(version range:==0.2.1)
| | +-six(version range:*)
| +-persisting-theory(version range:==0.2.1)
.
.
.
```

```
django-danceschool 0.8.6
.
.
.
| +-django-dynamic-preferences(version range:==1.7.0)
| | +-django(version range:>=1.11)
| | | +-pytz(version range:*)
| | +-persisting-theory(version range:==0.2.1)
| | +-six(version range:*)
| +-persisting-theory(version range:==0.2.1)
.
.
.
```

**Comment by EliotBerriot** [comment]
Hey @NeolithEra, thank you for bringing this issue to our attention. We can totaly loosen the requirement, and I tend to think switching to `persisting_theory>=0.2.1` is the easiest way to achieve this without breaking anything.

I'd be happy to review and merge a PR for this :)

**Comment by EliotBerriot** [comment]
Closed via #183
